src/assets/css/inputs.css

```diff
@@ -819,26 +818,48 @@ input:checked + .input-slider:before {
     background-color: $brand-color-primary;
     color: #fff;
 }
-.input-error .input-dropdown-list-option:hover,
-.input-error .input-dropdown-list-option:hover * {
-    color: $brand-font-color;
-    background-color: $brand-danger-color !important;
-}
-.input-error .input-dropdown-list-option.input-dropdown-list-option-key-on,
-.input-error .input-dropdown-list-option.input-dropdown-list-option-key-on * {
-    color: $brand-font-color;
-    background-color: $brand-danger-color !important;
+
+.input-error {
+    .raw-lookup-wrapper-bcg:hover {
+        background-color: $brand-danger-color !important;
+    }
+
+    .raw-lookup-wrapper-bcg:hover .input-dropdown-container .input-dropdown {
+        color: $brand-super-light-color;
+    }
+
+    .raw-lookup-wrapper-bcg:hover .input-dropdown,
+    .raw-lookup-wrapper-bcg:hover .input-dropdown input,
+    .raw-lookup-wrapper-bcg:hover .input-dropdown-container {
+        background-color: $brand-danger-color !important;
+        color: $brand-super-light-color;
+    }
+
+    .input-dropdown-list-option:hover,
+    .input-dropdown-list-option:hover * {
+        color: $brand-font-color;
+        background-color: $brand-danger-color !important;
+    }
+
+    .input-dropdown-list-option.input-dropdown-list-option-key-on,
+    .input-dropdown-list-option.input-dropdown-list-option-key-on * {
+        color: $brand-font-color;
+        background-color: $brand-danger-color !important;
+    }
```

Comment:
While you're at it, might as well use `&` to nest those

Reply:
Yeah the thing is I wasn't really focusing on cleaning up css here. And I'm reluctant to rewrite too much at once as it's easy to break things. So I'm just gonna keep fixing those things gradualy while I work on other stuff.

Reply:
Good point.

By the way, we should get source maps for CSS working. Otherwise it's really hard to find those composite class names